src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -4116,6 +4134,16 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <ClickOnceFiles Include="@(_FilesExcludedFromBundle)"/>
     </ItemGroup>
 
+    <!-- For single file publish in .net core app, sign the SF EXE if signing is enabled -->
+    <SignFile
+      CertificateThumbprint="$(_DeploymentResolvedManifestCertificateThumbprint)"
+      TimestampUrl="$(ManifestTimestampUrl)"
+      SigningTarget="$(PublishedSingleFilePath)"
```

Comment:
Would it be necessary to check if this path exists in SignFile's condition?

Reply:
I don't believe we need to check that because if the bundle EXE generation failed for any reason, we would fail in the manifest generation anyway and the first error in the build would be about the failure in GenerateSingleFileBundle.